src/Application.php

```diff
@@ -206,7 +191,6 @@ public function any($path, $middleware, $name = null)
      *
      * @param string|array|callable $path Either a URI path prefix, or middleware.
      * @param null|string|array|callable $middleware Middleware
-     * @return self
```

Comment:
It should not be removed, I think we should have here `$this` instead.
See PSR-5 draft (the very bottom of the page):
https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md
> `$this`, the element to which this type applies is the same exact instance as the current class in the given context. As such this type is a stricter version of static as, in addition, the returned instance must not only be of the same class but also the same instance.